mlrun/api/api/endpoints/marketplace.py

```diff
@@ -127,3 +158,14 @@ def get_object(
     if not ctype:
         ctype = "application/octet-stream"
     return Response(content=object_data, media_type=ctype)
+
+
+def _verify_action_allowed(auth_verifier: AuthVerifierDep, action: AuthorizationAction):
```

Comment:
I think you can add a `query_global_resource_permissions` to opa instead of this
WDYT ?

Reply:
Yes, sure, I'll do it.